rewrite-core/src/main/java/org/openrewrite/PrintOutputCapture.java

```diff
@@ -67,6 +71,42 @@ public PrintOutputCapture<P> clone() {
     }
 
     public interface MarkerPrinter {
+
+        @Incubating(since = "8.41.4")
+        @RequiredArgsConstructor
+        enum MarkerMode {
```

Comment:
I don't love the name. I think there is a similar design concept in `java.nio.charset`. There is a `Charset` abstract class and a separate top level class called `StandardCharsets`. I think a top level `StandardMarkerPrinters` might be a slightly better design.